salus-config.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/coinbase/salus/blob/master/docs/configuration.md
+
+# Used in the report to identify the project being scanned.
+project_name: gitops-secrets-nodejs
+
+# Defines where to send Salus reports and in what format.
+reports:
+  - uri: file://salus-report.txt
+    format: txt
+
+# All scanners to execute, or the String value "all"/"none"
+active_scanners:
+  - NPMAudit
+  - Semgrep
+
+# All scanners that will exit non-zero if they fail, or the String value "all"/"none"
+enforced_scanners: "all"
+
+scanner_configs:
+  Semgrep:
+    matches:
+      - config: ../semgrep_configs/eval.yaml
+        forbidden: true
+        exclude:
+          - node_modules
+          - bower_components
+          - external-src
```

Comment:
You can probably remove these excludes

Reply:
See https://github.com/DopplerUniversity/gitops-secrets-nodejs/pull/15